### DISCUSSION TITLE
Smart Image Crop

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#16332] [Feature Requests]
A nice thing would be to have a smart image crop enabled:

https://29a.ch/sandbox/2014/smartcrop/examples/testbed.html

I have used it in an app of mine already and it works realy nice.

![Download](https://user-images.githubusercontent.com/33295329/200127471-52115923-5ae8-4c8a-92e1-e5376202e3d2.png)
![Download (1)](https://user-images.githubusercontent.com/33295329/200127478-a6997802-c084-4ad2-bbb9-97c91733e6b4.png)

**Comment by azrikahar** [comment]
I believe this has been suggested in #10082, #7901, and #5659! Feel free to upvote #7901 as it's currently the one with the most upvotes (and so that the votes/discussions can converge in one place to make things easier to track) 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
